docs/source/configuration/settings-reference.md

```diff
@@ -177,7 +177,12 @@ contentMetadataTagsImageField
     The OpenGraph image that will represent this content item, will be used in the metadata HEAD tag as og:image for SEO purposes. Defaults to image. See the OpenGraph Protocol for more details.
 
 hasWorkingCopySupport
-    This setting will enable working copy support in your site. You need to install the `plone.app.iterate` add-on in your Plone site in order to make it working.
+    ```{deprecated} Volto 18.8.0
+
+    This setting is no longer used. Working copy support is now based on whether the `plone.app.iterate` add-on is installed in the backend.
```

Comment:
```suggestion
    This setting is unnecessary since Volto 18.8.0.
    Working copy support is now based on whether the `plone.app.iterate` add-on is installed in the backend.
```